### PR TITLE
Mbot fix paraview

### PR DESCRIPTION
**Pull request by nilsdeppe**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: cmake/FindParaView.cmake [hunk 1]
```diff
@@ -54,6 +54,17 @@ else()
...
+    ${PVPYTHON_BASEDIR}
+    "${PVPYTHON_BASEDIR}/lib*/python*/site-packages/paraview/simple.py"
+  )
+  find_file(PARAVIEW_PYTHONPATH
+    NAMES "simple.py"
+    PATHS "${PVPYTHON_BASEDIR}"
+    PATH_SUFFIXES "/.*"
+    REQUIRED)
+  # go back up to the site-packages
+  get_filename_component(PARAVIEW_PYTHONPATH ${PARAVIEW_PYTHONPATH} DIRECTORY)
```
Comment (kidder) [inline cmake/FindParaView.cmake:68]: is the duplication of this line intentional?

**Review by kidder — APPROVED** [review]
it works, but one question...

File: cmake/FindParaView.cmake (hunk 1 above)
Comment (nilsdeppe) [reply cmake/FindParaView.cmake:68]: Yes, because this finds `.../site-packages/paraview/simple.py` and we need ``../site-packages`

**Comment by knelli2** [comment]
@nilsvu Did you have a chance to look and test this yet?

**Comment by nilsvu** [comment]
No but if it fixes Mbot then it's fine. This branch of the cmake code doesn't run on CaltechHPC.

**Pull request merged**